src/gatling/simulations/scenarios/Solicitor_Bails.scala

```diff
@@ -0,0 +1,1383 @@
+
+    .group("XUI_Bails_410_Video_Link") {
+
+      exec(http("XUI_Bails_410_005_Video_Link")
+        .post("/data/case-types/Bail/validate?pageId=startApplicationapplicantVideoHearingDetails")
+        .headers(Headers.commonHeader)
+        .header("accept", "application/vnd.uk.gov.hmcts.ccd-data-store-api.case-data-validate.v2+json;charset=UTF-8")
+        .header("x-xsrf-token", "${XSRFToken}")
+        .body(ElFileBody("bodies/bails/BailsVideoLink.json"))
+        .check(substring("videoHearing1")))
+
+    }
+    .pause(MinThinkTime, MaxThinkTime)
+
+
+    /*==================================================================================================
+    * Enter the legal representative’s details
+    ====================================================================================================*/
+
+    .group("XUI_Bails_420_Legal_Reps_Details") {
+
+      exec(http("XUI_Bails_420_005_Legal_Reps_Details")
+        .post("/data/case-types/Bail/validate?pageId=startApplicationlegalRepresentativeDetails")
+        .headers(Headers.commonHeader)
+        .header("accept", "application/vnd.uk.gov.hmcts.ccd-data-store-api.case-data-validate.v2+json;charset=UTF-8")
+        .header("x-xsrf-token", "${XSRFToken}")
+        .body(ElFileBody("bodies/bails/BailsLegalRepsDetails.json"))
+        .check(substring("isLegalRep")))
+
+    }
+    .pause(MinThinkTime, MaxThinkTime)
+
+
+    /*==================================================================================================
+    * Check your answers
+    ====================================================================================================*/
+
+    .group("XUI_Bails_430_Check_Your_Answers") {
+
+      exec(http("XUI_Bails_430_005_Check_Your_Answers")
+        .post(BaseURL + "/data/case-types/Bail/cases?ignore-warning=false")
+        .headers(Headers.commonHeader)
+        .header("accept", "application/vnd.uk.gov.hmcts.ccd-data-store-api.create-case.v2+json;charset=UTF-8")
+        .header("x-xsrf-token", "${XSRFToken}")
+        .body(ElFileBody("bodies/bails/BailsCheckYourAnswers.json"))
+        .check(jsonPath("$.id").saveAs("case_Id"))
+        .check(jsonPath("$.callback_response_status").is("CALLBACK_COMPLETED")))
+
+    }
+    .pause(MinThinkTime, MaxThinkTime)
+
+
+  val SubmitBailApplication =
+
+    /*======================================================================================
+    * Load the newly created case
+    ======================================================================================*/
+
+    group("XUI_Bails_440_Open_Case") {
+
+      exec(Common.healthcheck("%2Fcases%2Fcase-details%2F${case_Id}"))
+
+      .exec(http("XUI_Bails_440_005_CreateCase")
+        .get("/data/internal/cases/${case_Id}")
+        .headers(Headers.commonHeader)
+        .header("accept", "application/vnd.uk.gov.hmcts.ccd-data-store-api.ui-case-view.v2+json")
+        .check(jsonPath("$.case_id").is("${case_Id}")))
```

Comment:
maybe check the $.state field if useful

Reply:
Changed it to check the state.name is 'Application started'